packages/cluster-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cockroachlabs/cluster-ui",
-  "version": "0.2.15",
```

Comment:
@koorosh When this PR is merged, will the Github Action publish version `0.2.43` or will it bump the version and publish `0.2.44`?

Reply:
It will increment the version (`0.2.44`)

Reply:
GH Actions will publish `0.2.44`. I will include version `0.2.43` in the deprecated versions to avoid future confusion.